### DISCUSSION TITLE
Open deck editor malfunction

### DISCUSSION THREAD

**Discussion by willmodelisme** [shanteacontrols/OpenDeck#262] [General]
Hello all and happy new year.
I randomly have trouble with the Opendeck configurator, online or app, that is unable to recognize my M board.
In the top right corner I have that info  BoardDubFocus v1Firmwarevundefined.undefined.undefined.
I have to do a factory reset and reload my saved configuration file.
Any tips about that ?

**Comment by paradajz** [comment]
People have mentioned this few times but I was sadly never able to reproduce, so, cannot really help you. My guess is that it's related to either Chrome or the configurator tool, not the firmware itself.

**Reply by paradajz** [reply]
But please ensure you're using the latest firmware.

**Reply by squaretones** [reply]
Could you point me to where to find instructions on how to flash the latest firmware onto an OpenDeck M board? I gather it's a RP2040 board but the page with instructions for flashing RP2040 boards doesn't mention the OpenDeck board. Thanks!
